WikiDrips/WikiTableViewController.swift

```diff
@@ -50,12 +47,55 @@ class WikiTableViewController: UITableViewController, UISearchResultsUpdating {
             guard let strongSelf = self else { return }
             DispatchQueue.main.async(){
```

Comment:
This Dispatch is also a problem. If a user is typing really fast do you really want to be kicking off a bunch of requests back to back? I think at the very least you would want:
1. If a new request comes in, cancel the old `WikiRequest`

As a bonus exercice for next PR, you may also want to consolidate keystrokes so that search is only kicked off after say 400 ms. Note though, that consolidating keystrokes is NOT necessary to fix you asynchronous issues.